services/wireguard/endpoint/kernelspace/client.go

```diff
@@ -132,3 +166,10 @@ func stringToKey(key string) (wgtypes.Key, error) {
 	}
 	return wgtypes.NewKey(k)
 }
+
+func ipExec(args ...string) error {
+	if err := exec.Command("ip", args...).Run(); err != nil {
```

Comment:
We prob will need to use exec.Command().CombinedOutput here since simple run will swallow error outputs and there will be no way to test what the problem was.

Reply:
Each external command requiring superuser privileges we run with sudo. `/sbin/ip` is included in linux sudoers.

Reply:
Fixed.